### ISSUE TITLE
hyperlink formatting issues

### ISSUE THREAD

**Issue by etdsoft** [securityroots/dradispro-tracker#244]
> -When a link includes brackets, they don't show up in the text. The link target is still correct.
> -If there is a colon after the link, it is duplicated in the report.
> -When there is a link or bolded text inside a listing, the space before is truncated. Without the listing these work as expected. This probably goes for other formattings besides just bold and link

Originally reported by Jarmo.

```
#[Description]#
https://test.com?alert(document.domain)

https://test.com?double_colon:

* crammed https://test.com?test hyperlink
* crammed *bold* stuff

```

![image002](https://cloud.githubusercontent.com/assets/53006/8353208/e06ea21c-1b3b-11e5-9b30-53f503f67c6e.png)

**Comment by etdsoft** [comment]
1. The parenthesis are being interpreted as URL's `:title`. Inquiring upstream for a workaround:

https://jgarber.lighthouseapp.com/projects/13054-redcloth/tickets/279-links-with-a-name-that-contains-a-trailing-whatever

**Comment by etdsoft** [comment]
3) is now fixed and will ship in v2.0

**Comment by rachkor** [comment]
Closing, moving over to https://github.com/dradis/dradis-ce/issues/174. Please follow there for updates
